adapters/oidc/spring-boot/src/main/java/org/keycloak/adapters/springboot/KeycloakSpringBootConfiguration.java

```diff
@@ -209,7 +209,8 @@ public void customize(Server server) {
                 }
             }
 
-            WebAppContext webAppContext = server.getBean(WebAppContext.class);
+            WebAppContext webAppContext = (WebAppContext) server.getHandler();;
```

Comment:
yeah for the mutliple context I'm not sure how that applies with using the embedded springboot stuff ..
And thx for the semicolon and empty line, I think it's time I call it a day ;)